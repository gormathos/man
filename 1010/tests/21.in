sol_id;user_id;task_id;lang_id;score
116412;1011;2002;5;18.75
116413;1047;2005;5;45.24
116415;1050;2001;2;100.00
116418;1092;2004;1;100.00
116422;1092;2003;4;96.30
116427;1075;2003;4;88.89
116433;1010;2001;3;34.78
116440;1078;2004;1;61.90
116448;1004;2005;1;83.33
116457;1046;2001;5;86.96
116467;1064;2001;2;100.00
116478;1061;2001;1;100.00
116490;1084;2004;3;33.33
116503;1090;2004;5;0.00
116517;1042;2001;5;73.91
116532;1099;2003;2;0.00
116548;1065;2002;1;18.75
116565;1095;2001;5;100.00
116583;1059;2004;1;52.38
116602;1040;2002;5;100.00
116622;1071;2002;4;0.00
116643;1095;2001;3;100.00
116665;1011;2002;2;0.00
116688;1005;2002;1;6.25
116712;1085;2001;2;60.87
116737;1066;2005;4;0.00
116763;1096;2004;5;21.43
116790;1079;2005;3;40.48
116818;1031;2002;5;100.00
116847;1087;2001;5;100.00
116877;1070;2004;3;30.95
116908;1047;2001;3;52.17
116940;1021;2004;5;19.05
116973;1004;2005;1;28.57
117007;1003;2001;5;100.00
117042;1070;2005;5;11.90
117078;1077;2003;2;14.81
117115;1037;2001;1;0.00
117153;1098;2004;3;100.00
117192;1078;2002;4;53.12
117232;1049;2005;2;78.57
117273;1073;2002;3;100.00
117315;1091;2002;4;100.00
117358;1058;2002;5;31.25
117402;1023;2004;1;0.00
117447;1074;2003;5;29.63
117493;1044;2005;3;97.62
117540;1081;2004;1;0.00
117588;1084;2002;3;75.00
117637;1076;2001;2;100.00
117687;1005;2001;4;65.22
117738;1067;2003;5;88.89
117790;1037;2003;2;100.00
117843;1017;2003;5;3.70
117897;1072;2005;5;0.00
117952;1041;2002;3;93.75
118008;1095;2001;5;0.00
118065;1070;2002;2;78.12
118123;1048;2004;3;40.48
118182;1032;2002;1;100.00
118242;1015;2004;5;26.19
118303;1003;2003;3;37.04
118365;1049;2004;4;0.00
118428;1009;2005;3;19.05
118492;1017;2005;2;0.00
118557;1014;2001;2;91.30
118623;1053;2002;1;100.00
118690;1041;2004;3;0.00
118758;1012;2002;1;0.00
118827;1084;2002;2;78.12
118897;1023;2003;1;33.33
118968;1018;2004;5;83.33
119040;1022;2003;4;100.00
119113;1033;2005;2;66.67
119187;1099;2001;1;39.13
119262;1036;2003;3;100.00
119338;1092;2005;1;9.52
119415;1047;2003;3;0.00
119493;1096;2001;3;13.04
119572;1072;2001;3;100.00
119652;1024;2005;4;100.00
119733;1022;2003;3;100.00
119815;1099;2005;4;0.00
119898;1074;2002;2;12.50
119982;1028;2004;1;97.62
120067;1085;2005;1;14.29
120153;1017;2003;4;100.00
120240;1054;2005;4;0.00
120328;1065;2005;2;47.62
120417;1018;2001;2;8.70
120507;1041;2004;5;0.00
120598;1042;2003;2;0.00
120690;1096;2001;5;100.00
120783;1089;2005;1;80.95
120877;1077;2004;2;100.00
120972;1065;2003;4;0.00
121068;1017;2004;5;88.10
121165;1089;2004;1;0.00
121263;1036;2005;2;14.29
121362;1070;2004;4;0.00
121462;1047;2001;3;78.26
121563;1008;2004;4;100.00
121665;1085;2004;3;0.00
121768;1036;2002;5;0.00
121872;1053;2005;5;0.00
121977;1018;2002;2;0.00
122083;1076;2005;2;0.00
122190;1014;2004;1;0.00
122298;1062;2002;2;0.00
122407;1073;2002;3;0.00
122517;1044;2001;3;0.00
122628;1084;2005;1;35.71
122740;1029;2002;1;0.00
122853;1042;2003;1;29.63
122967;1047;2003;3;37.04
123082;1047;2004;1;66.67
123198;1012;2004;5;16.67
123315;1013;2002;4;100.00
123433;1008;2004;5;21.43
123552;1028;2003;5;100.00
123672;1096;2001;1;100.00
123793;1088;2002;2;100.00
123915;1082;2005;5;80.95
124038;1089;2002;5;0.00
124162;1034;2004;1;0.00
124287;1068;2005;4;52.38
124413;1097;2002;3;0.00
124540;1005;2005;2;100.00
124668;1078;2005;4;52.38
124797;1057;2003;3;51.85
124927;1088;2005;3;100.00
125058;1095;2004;4;0.00
125190;1071;2003;5;37.04
125323;1053;2001;1;47.83
125457;1012;2004;3;23.81
125592;1066;2002;4;43.75
125728;1009;2005;2;38.10
125865;1068;2003;1;55.56
126003;1070;2003;2;92.59
126142;1018;2003;2;100.00
126282;1057;2004;1;0.00
126423;1081;2005;2;0.00
126565;1049;2001;1;4.35
126708;1093;2003;2;100.00
126852;1014;2004;3;0.00
126997;1042;2003;4;66.67
127143;1095;2001;3;100.00
127290;1081;2003;1;0.00
127438;1033;2004;3;61.90
127587;1095;2005;3;28.57
127737;1079;2003;5;81.48
127888;1095;2003;5;0.00
128040;1040;2002;5;78.12
128193;1074;2003;1;0.00
128347;1009;2002;2;31.25
128502;1033;2003;2;100.00
128658;1069;2001;4;100.00
128815;1072;2001;3;95.65
128973;1043;2003;2;44.44
129132;1042;2003;5;100.00
129292;1017;2005;5;28.57
129453;1089;2002;5;75.00
129615;1092;2005;5;100.00
129778;1054;2004;5;100.00
129942;1041;2001;2;52.17
130107;1055;2004;1;100.00
130273;1076;2004;1;35.71
130440;1038;2002;1;40.62
130608;1041;2001;2;0.00
130777;1043;2003;3;74.07
130947;1058;2003;4;22.22
131118;1079;2004;3;0.00
131290;1056;2005;2;100.00
131463;1073;2004;1;33.33
131637;1019;2003;1;77.78
131812;1082;2001;1;100.00
131988;1045;2003;1;100.00
132165;1011;2002;5;15.62
132343;1092;2001;3;100.00
132522;1096;2004;4;0.00
132702;1023;2002;3;0.00
132883;1057;2002;4;0.00
133065;1060;2003;2;18.52
133248;1003;2003;3;0.00
133432;1057;2002;1;65.62
133617;1082;2002;1;0.00
133803;1021;2004;2;100.00
133990;1034;2004;5;0.00
134178;1001;2002;3;9.38
134367;1028;2004;5;100.00
134557;1082;2005;2;0.00
134748;1093;2002;2;100.00
134940;1028;2001;4;0.00
135133;1035;2002;5;15.62
135327;1018;2005;4;100.00
135522;1005;2001;2;0.00
135718;1067;2003;5;0.00
135915;1034;2001;5;100.00
136113;1009;2005;1;45.24
136312;1063;2005;4;76.19
136512;1007;2003;1;66.67
136713;1075;2005;3;19.05
136915;1062;2003;2;3.70
137118;1064;2001;4;100.00
137322;1009;2003;2;3.70
137527;1018;2002;1;100.00
137733;1062;2001;5;0.00
137940;1017;2001;2;100.00
138148;1041;2004;1;50.00
138357;1010;2004;3;23.81
138567;1055;2001;2;34.78
138778;1072;2004;2;69.05
138990;1042;2002;1;12.50
139203;1053;2002;4;34.38
139417;1097;2002;2;0.00
139632;1004;2004;2;100.00
139848;1083;2001;5;100.00
140065;1067;2003;2;100.00
140283;1053;2004;1;38.10
140502;1036;2002;3;56.25
140722;1091;2003;2;0.00
140943;1002;2002;2;0.00
141165;1021;2004;3;100.00
141388;1036;2001;1;86.96
141612;1044;2003;5;29.63
141837;1034;2005;4;73.81
142063;1005;2002;5;9.38
142290;1005;2003;1;66.67
142518;1063;2005;4;97.62
142747;1038;2005;2;100.00
142977;1007;2004;5;0.00
143208;1028;2001;3;0.00
143440;1049;2002;4;0.00
143673;1048;2002;3;0.00
143907;1087;2002;4;0.00
144142;1071;2004;3;100.00
144378;1099;2004;2;4.76
144615;1016;2005;2;0.00
144853;1075;2003;1;92.59
145092;1061;2005;1;57.14
145332;1046;2001;5;52.17
145573;1037;2005;3;0.00
145815;1034;2002;2;0.00
146058;1097;2003;1;66.67
146302;1094;2002;2;100.00
146547;1073;2001;3;82.61
146793;1096;2002;3;96.88
147040;1094;2001;2;60.87
147288;1070;2003;3;0.00
147537;1048;2003;5;37.04
147787;1083;2003;3;70.37
148038;1036;2002;4;3.12
148290;1033;2002;4;0.00
148543;1077;2001;1;100.00
148797;1054;2003;2;96.30
149052;1024;2002;1;0.00
149308;1050;2003;5;96.30
149565;1090;2004;4;11.90
149823;1065;2002;5;100.00
150082;1079;2001;4;82.61
150342;1055;2001;4;100.00
150603;1035;2001;3;100.00
150865;1090;2002;2;84.38
151128;1047;2001;3;17.39
151392;1017;2001;2;0.00
151657;1074;2001;5;13.04
151923;1061;2004;5;83.33
152190;1063;2005;1;100.00
152458;1065;2001;5;60.87
152727;1071;2005;1;100.00
152997;1088;2002;4;100.00
153268;1034;2001;1;100.00
153540;1092;2003;3;7.41
153813;1025;2002;3;50.00
154087;1050;2005;3;88.10
154362;1003;2004;3;0.00
154638;1071;2003;5;100.00
154915;1091;2002;5;0.00
155193;1095;2005;4;100.00
155472;1085;2001;4;86.96
155752;1098;2004;1;0.00
156033;1095;2001;1;43.48
156315;1017;2005;2;100.00
156598;1048;2004;4;0.00
156882;1028;2005;4;0.00
157167;1100;2002;4;9.38
157453;1008;2003;5;62.96
157740;1045;2003;1;0.00
158028;1028;2001;1;69.57
158317;1059;2005;1;100.00
158607;1070;2005;2;100.00
158898;1089;2005;2;100.00
159190;1065;2004;3;69.05
159483;1053;2004;5;59.52
159777;1030;2001;4;100.00
160072;1074;2001;4;65.22
160368;1021;2004;2;100.00
160665;1013;2004;3;45.24
160963;1080;2001;2;43.48
161262;1063;2005;5;16.67
161562;1005;2004;1;28.57
161863;1052;2003;3;55.56
162165;1054;2003;3;100.00
162468;1071;2003;1;74.07
162772;1006;2004;5;100.00
163077;1011;2001;5;86.96
163383;1020;2001;4;69.57
163690;1054;2003;1;48.15
163998;1001;2005;4;78.57
164307;1029;2004;5;100.00
164617;1064;2004;3;100.00
164928;1077;2005;4;47.62
165240;1091;2005;5;38.10
165553;1070;2003;4;66.67
165867;1041;2005;3;100.00
166182;1077;2002;3;56.25
166498;1096;2002;1;100.00
166815;1081;2002;4;0.00
167133;1055;2004;5;57.14
167452;1008;2004;4;0.00
167772;1051;2001;2;0.00
168093;1085;2004;5;100.00
168415;1100;2002;1;81.25
168738;1089;2004;1;50.00
169062;1071;2001;1;100.00
169387;1048;2005;5;100.00
169713;1075;2003;5;96.30
170040;1063;2001;4;8.70
170368;1011;2003;2;100.00
170697;1089;2005;2;83.33
171027;1092;2003;2;0.00
171358;1009;2001;2;100.00
171690;1038;2005;4;0.00
172023;1025;2001;5;100.00
172357;1089;2005;5;9.52
172692;1094;2005;1;85.71
173028;1022;2003;4;100.00
173365;1057;2001;1;0.00
173703;1066;2005;3;0.00
174042;1058;2002;3;100.00
174382;1060;2001;1;0.00
174723;1029;2003;1;100.00
175065;1097;2003;1;77.78
175408;1034;2003;3;0.00
175752;1021;2005;2;0.00
176097;1043;2001;5;86.96
176443;1034;2001;1;52.17
176790;1043;2003;5;100.00
177138;1017;2003;4;0.00
177487;1035;2002;4;37.50
177837;1014;2001;1;100.00
178188;1059;2005;1;100.00
178540;1099;2001;4;0.00
178893;1083;2003;2;3.70
179247;1035;2002;3;93.75
179602;1001;2004;4;9.52
179958;1094;2004;2;59.52
180315;1096;2002;3;0.00
180673;1069;2005;3;100.00
181032;1080;2001;1;0.00
181392;1093;2005;4;100.00
181753;1031;2003;3;0.00
182115;1049;2005;4;30.95
182478;1007;2003;5;100.00
182842;1051;2005;3;26.19
183207;1083;2001;3;100.00
183573;1058;2003;4;0.00
183940;1078;2002;1;0.00
