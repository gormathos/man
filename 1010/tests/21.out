Pascal: 53 81 19 19
Python: 49 73 24 20
C++: 49 74 18 19
Java: 49 64 15 22
C#: 52 76 22 13
